pkg/devfile/parser/data/common/command_helper.go

```diff
@@ -0,0 +1,71 @@
+package common
+
+import (
+	"strings"
+
+	v1 "github.com/devfile/api/pkg/apis/workspaces/v1alpha2"
+)
+
+// GetID returns the ID of the command
+func GetID(dc v1.Command) string {
+	return dc.Id
```

Comment:
Since `dc.Id` is exported, a getter is not needed (unless there's a plan to do some actual processing here in the future).